webservice/app/Http/Controllers/AuthController.php

```diff
@@ -93,6 +92,31 @@ public function sendLockoutResponse(Request $request)
         return $this->responseWithTooManyRequests($message);
     }
 
+    /**
+     * Revoke the user's token.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function revokeToken()
+    {
+        Auth::guard('api')->logout();
+
+        return $this->responseWithNoContent();
+    }
+
+    /**
+     * Refresh the user's token.
+     *
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function refreshToken(Request $request)
+    {
+        $token = Auth::guard('api')->refresh();
+
+        return $this->response(compact('token'));
```

Comment:
What is gonna be returned when token cannot be refreshed? I understand there's a constraint of how many times a token can be refreshed.